roles/load_aware_undeploy_trimaran/tasks/main.yml

```diff
@@ -0,0 +1,11 @@
+- name: Delete Trimaran namespace
+  shell:
+    oc delete ns trimaran
+  
+- name: Delete user monitoring namespace
+  shell:
+    oc delete ns openshift-user-workload-monitoring 
```

Comment:
you cannot delete this namespace, we don't know if we created it
let this part (the activation of the user-workload-monitoring) untouched, it's too complex to handle properly

Reply:
You're right. I meant to delete the config map that we purposefully create to setup user application monitoring

https://github.com/openshift-psap/ci-artifacts/blob/main/roles/load_aware_deploy_trimaran/files/cluster-monitoring-config.yaml